docs/src/pages/premium-themes/gmail/modules/layout.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import classNames from 'classnames';
+import { makeStyles, useTheme } from '@material-ui/styles';
+import Drawer from '@material-ui/core/Drawer';
+import Fab from '@material-ui/core/Fab';
+import Avatar from '@material-ui/core/Avatar';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import InboxIcon from '@material-ui/icons/MoveToInbox';
+import MailIcon from '@material-ui/icons/Mail';
+import Add from '@material-ui/icons/Add';
+import Settings from '@material-ui/icons/Settings';
+import Gsearch from './components/search';
```

Comment:
Can we make the exported name match the file name? It makes the component easier to open in the text editor. It's also a popular convention.

```suggestion
import Gsearch from './components/Gsearch';
```